docs/data/pages.ts

```diff
@@ -533,7 +533,16 @@ const pages: MuiPage[] = [
             title: 'Heatmap',
             plan: 'pro',
           },
-          { pathname: '/x/react-charts/funnel', title: 'Funnel', plan: 'pro', unstable: true },
+          {
+            pathname: '/x/react-charts-funnel',
+            title: 'Funnel',
+            plan: 'pro',
+            unstable: true,
+            children: [
+              { pathname: '/x/react-charts/funnel', title: 'Funnel Overview' },
+              { pathname: '/x/react-charts/pyramid', title: 'Pyramid demo' },
```

Comment:
```suggestion
              { pathname: '/x/react-charts/pyramid', title: 'Pyramid chart' },
```

Reply:
ah yeah, I had seen this earlier but forgot to change/comment.

Reply:
> It currently follows the pattern

Yeah, I'm aware. It's just that I remember we discussing that we should move away from demo pages and have everything in a single page, when possible. 

It probably makes sense to rename the area demo page into area chart and add more context.

Reply:
> It probably makes sense to rename the area demo page into area chart and add more context.

For now, I think we can follow the current structure and do this in a different PR.